international-standard/document.adoc

```diff
@@ -1,35 +1,134 @@
 = ISO Codes for timekeeping systems
+:comment: ### See comment 1 below
 :docnumber: 99999
+:comment: ### Document number assigned by the Technical committee. See comment 1
 :tc-document-number: 9999
+:comment: ### Document edition
 :edition: 1
+:comment: ### The year of copyright
 :copyright-year: 2029
-:language: en
+:comment: ### Introductory component of the English title of the document. 
 :title-intro-en: Date and time
+:comment: ### Main component of the English title of the document (mandatory)
 :title-main-en: Codes for timekeeping systems
+:comment: ### Introductory component of the French title of the document.
 :title-intro-fr: Date et l'heure
+:comment: ###  Main component of the French title of the document (mandatory)
 :title-main-fr: Codes pour les systèmes de chronométrage
-:doctype: standard
+:comment: ### See comment 2 below
+:doctype: international-standard
+:comment: ### See comment 3 below
 :docstage: 00
+:comment: ### See comment 4 below
 :docsubstage: 00
+:comment: ### See comment 5 below
 :technical-committee-type: TC
+:comment: ### Number of the relevant ISO subcommittee
 :technical-committee-number: 154
+:comment: ### Name of the relevant ISO technical committee (mandatory)
 :technical-committee: Processes, data elements and documents in commerce, industry and administration
+:comment: ### See comment 6 below
 :workgroup-type: WG
+:comment: ### Number of the relevant ISO workgroup
 :workgroup-number: 5
+:comment: ### Name of the relevant ISO workgroup
 :workgroup: Date and time
+:comment: ### Secretariat example values: ANSI, SAC, ACA...
 :secretariat: SAC
+:comment: ### Sets document language. Two values are permitted: en, fr.
 :language: en
+:comment: ### Draft status, enables display of reviewer notes; remove if final document
 :draft:
+:comment: ### Enables table of contents
 :toc:
+:comment: ### Enables stem math mode
 :stem:
+:comment: ### Sets crossreference style
 :xrefstyle: short
```

Comment:
@opoudjis I think we should make this default and remove it from template attributes.

Reply:
Ok, removing...